chrome/content/zotero/selectItemsDialog.js

```diff
@@ -60,7 +60,7 @@ var doLoad = async function () {
 				onItemSelected();
 			}
 		},
-		id: "select-items-dialog",
+		id: io.treeID || "select-items-dialog",
```

Comment:
This is not going to work for views that import `selectItemsDialog.js` as part of their code. Also you need to change the code that uses this code to actually provide the IDs. And the variable needs to change to `itemTreeID`, since some views, like editBibliography also have a collection tree.